vite.js

```diff
@@ -81,9 +81,7 @@ module.exports = function style9Plugin(opts = {}) {
       return null;
     },
     transform(code, id) {
-      if (!filter(id)) {
-        return;
-      }
+      if (!filter(id) && !/style9/.test(code)) return;
```

Comment:
this should be `||`

Reply:
why use `||`. If code don't have `style9` it will be skip. isn't it?

Reply:
my mistake.